types/react/index.d.ts

```diff
@@ -2913,6 +2913,7 @@ declare namespace React {
 
     interface ImgHTMLAttributes<T> extends HTMLAttributes<T> {
         alt?: string;
+        crossOrigin?: "anonymous" | "use-credentials" | "";
```

Comment:
This change probably should be done to all instances of `crossOrigin` in the definitions